### PR TITLE
Check that spherical shell inner radius is less than outer radius

### PR DESCRIPTION
**Pull request by bartniday**

### CODE REVIEW

**Comment by gassmoeller** [comment]
Yes, that makes sense :+1:

File: source/geometry_model/spherical_shell.cc [hunk 1]
```diff
@@ -396,7 +396,6 @@ namespace aspect
                              "this geometry are 90, 180, and 360 in 2d; "
                              "and 90 and 360 in 3d. "
                              "Units: degrees.");
-
```
Comment (MFraters) [inline source/geometry_model/spherical_shell.cc]: why are you removing this line?

Comment (MFraters) [reply source/geometry_model/spherical_shell.cc]: hmm, interesting. I never realized we (also me if I look at other models) did that :s

**Pull request merged**